testutil/key_test.go

```diff
@@ -7,19 +7,18 @@ import (
 
 	"github.com/line/lbm-sdk/crypto/hd"
 	"github.com/line/lbm-sdk/crypto/keyring"
-	"github.com/line/lbm-sdk/server"
-	"github.com/line/lbm-sdk/types"
+	sdk "github.com/line/lbm-sdk/types"
```

Comment:
There is no other reason, but it is used as an alias in most uses.

Reply:
The original code was not using the alias, and AFAIK it's also common not to use aliases on types package in the files which do not belong to any modules.
So I wanted to tell you if you use the alias here, we need to take special care on merging this file later, because we have a different version of it.

Reply:
I removed that alias. Thx for your good opinion :)